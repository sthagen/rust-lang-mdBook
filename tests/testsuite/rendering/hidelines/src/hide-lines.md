# Hide Lines

```python
~hidden()
nothidden():
~    hidden()
    ~hidden()
    nothidden()
```

```python,hidelines=!!!
!!!hidden()
nothidden():
!!!    hidden()
    !!!hidden()
    nothidden()
```

```rust
#![allow(something)]
#
#hidden();
# hidden();
## not_hidden();
#[not_hidden]
not_hidden();
```
