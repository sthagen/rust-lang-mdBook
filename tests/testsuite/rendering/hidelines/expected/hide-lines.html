<h1 id="hide-lines"><a class="header" href="#hide-lines">Hide Lines</a></h1>
<pre><code class="language-python"><span class="boring">hidden()
</span>nothidden():
<span class="boring">    hidden()
</span><span class="boring">    hidden()
</span>    nothidden()
</code></pre>
<pre><code class="language-python hidelines=!!!"><span class="boring">hidden()
</span>nothidden():
<span class="boring">    hidden()
</span><span class="boring">    hidden()
</span>    nothidden()
</code></pre>
<pre class="playground"><code class="language-rust"><span class="boring">#![allow(unused)]
</span>#![allow(something)]
<span class="boring">fn main() {
</span><span class="boring">
</span>#hidden();
<span class="boring">hidden();
</span># not_hidden();
#[not_hidden]
not_hidden();
<span class="boring">}</span></code></pre>